configs/localConfig.json

```diff
@@ -725,52 +729,248 @@
+        {
+          "name": "Maps",
+          "cfg": {
+            "mapsOptions": {
+              "start": 0,
+              "limit": 12
+            },
+            "fluid": true
+          }
+        },
+        {
+          "name": "Dashboards",
+          "cfg": {
+            "mapsOptions": {
+              "start": 0,
+              "limit": 12
+            },
+            "fluid": true
+          }
+        },
+        {
+          "name": "GeoStories",
+          "cfg": {
+            "mapsOptions": {
+              "start": 0,
+              "limit": 12
+            },
+            "fluid": true
+          }
+        },
+        {
+          "name": "Contexts",
+          "cfg": {
+            "mapsOptions": {
+              "start": 0,
+              "limit": 12
+            },
+            "fluid": true
+          }
+        }],
+      "dashboard": [{
+        "name": "OmniBar",
+        "cfg": {
+          "containerPosition": "header",
+          "className": "navbar shadow navbar-home"
+        }
+      }, "Login", "Language", "NavMenu", "DashboardSave", "DashboardSaveAs", "Attribution", "Home", {
+        "name": "Share",
+        "cfg": {
+          "showAPI": false,
+          "advancedSettings": false,
+          "shareUrlRegex": "(h[^#]*)#\\/dashboard\\/([A-Za-z0-9]*)",
+          "shareUrlReplaceString": "$1dashboard-embedded.html#/$2",
+          "embedOptions": {
+            "showTOCToggle": false,
+            "showConnectionsParamToggle": true,
+            "allowFullScreen": false,
+            "sizeOptions": {
+              "Small": { "width": 600, "height": 500 },
+              "Medium": { "width": 800, "height": 600},
+              "Large": { "width": 1000, "height": 800},
+              "Custom": {"width": 0, "height": 0}
+            },
+            "selectedOption": "Small"
+          }
+        }
+      },
+        {
+          "name": "DashboardEditor",
+          "cfg": {
+            "selectedService": "GeoSolutions GeoServer CSW",
+            "services": {
+              "GeoSolutions GeoServer CSW": {
+                "url": "https://gs-stable.geo-solutions.it/geoserver/csw",
+                "type": "csw",
+                "title": "GeoSolutions GeoServer CSW"
               },
-              "fluid": true
-            }
-          },
-          "MapSearch",
-          "CreateNewMap",
-          "FeaturedMaps"],
-        "admin": [{
-          "name": "Header",
-            "cfg": {
-              "page": "msadmin"
+              "GeoSolutions GeoServer WMS": {
+                "url": "https://gs-stable.geo-solutions.it/geoserver/wms",
+                "type": "wms",
+                "title": "GeoSolutions GeoServer WMS"
+              },
+              "GeoSolutions GeoServer WMTS": {
+                "url": "https://gs-stable.geo-solutions.it/geoserver/gwc/service/wmts",
+                "type": "wmts",
+                "title": "GeoSolutions GeoServer WMTS"
+              }
+            },
```

Comment:
Here we should leave it empty or use the services configured in geOrchestra. @tdipisa what services the dashboard should have as default in geOrchestra default installation? local geoserver (csw extension is not present, so we may try to use wfs or so...)

Reply:
> Here we should leave it empty or use the services configured in geOrchestra. @tdipisa what services the dashboard should have as default in geOrchestra default installation? local geoserver (csw extension is not present, so we may try to use wfs or so...)

@alexander-fedorenko @offtherailz please use the same have been always used until now for the [default context](https://github.com/georchestra/mapstore2-georchestra/blob/master/configs/localConfig.json#L111)

Reply:
> @tdipisa They said that home page is optional. This means that the button that links to the home from the main map should not be included by default, isn't it?

It is correct

Reply:
@alexander-fedorenko please proceed applying requested changes.

Reply:
> > @tdipisa They said that home page is optional. This means that the button that links to the home from the main map should not be included by default, isn't it?
> 
> Of course, it is correct.

chiming in, anyway its just a matter of adding the 'Home' plugin to the context/default config, right ?